configurator/frontend/src/types.d/destination.d.ts

```diff
@@ -1,4 +1,4 @@
-declare type DestinationType = 'postgres' | 'bigquery' | 'redshift' | 'clickhouse' | 'snowflake' | 'facebook' | 'google_analytics' | 'webhook';
+declare type DestinationType = 'postgres' | 'bigquery' | 'redshift' | 'clickhouse' | 'snowflake' | 'facebook' | 'google_analytics' | 'webhook' | 'mySQL';
```

Comment:
MySQL destination type should be a lower cased constant: `mysql`

Reply:
replaced with lower cased constant (`mysql`)